octomap_server/CMakeLists.txt

```diff
@@ -73,6 +73,9 @@ target_link_libraries(octomap_tracking_server_node ${PROJECT_NAME} ${LINK_LIBS})
 add_library(octomap_server_nodelet src/octomap_server_nodelet.cpp)
 target_link_libraries(octomap_server_nodelet ${PROJECT_NAME} ${LINK_LIBS})
 
+add_library(octomap_color_server_nodelet src/octomap_server_nodelet.cpp)
```

Comment:
Thank you for your contribution. Can you please also add this nodelet to the `install` command below?

Thank you!